.github/workflows/ci.yml

```diff
@@ -115,9 +137,49 @@ jobs:
         vtk_version: ${{matrix.vtk_version}}
         raytracing_label: ${{matrix.raytracing_label}}
         exclude_deprecated_label: ${{matrix.exclude_deprecated_label}}
-        bundle_label: ${{matrix.bundle_label}}
         egl_label: ${{matrix.egl_label}}
 
+#----------------------------------------------------------------------------
+# MacOS CI: Build and test, cross-vtk build matrix
+#----------------------------------------------------------------------------
+  macos:
+    if: github.event.pull_request.draft == false
+    needs: cache_lfs
+
+    strategy:
+      fail-fast: false
+      matrix:
+        vtk_version: [commit, v9.2.6, v9.1.0, v9.0.0]
+        cpu: [x86_64, arm64]
+        bundle_label: [no-bundle]
+        include:
+          - vtk_version: commit
+            bundle_label: bundle
+            cpu: x86_64
+
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: 'source'
+        fetch-depth: 0
+        lfs: false
+
+    - name: Copy LFS Data
+      uses: ./source/.github/actions/lfs-copy
+      with:
+        lfs_sha: ${{ needs.cache_lfs.outputs.lfs_sha}}
+        workflow_label: 'consumer'
```

Comment:
we could consider putting that into the action that needs it instead of duplcating it.